ENIGMAsystem/SHELL/Platforms/SDL/Event.cpp

```diff
@@ -208,11 +208,10 @@ void SDL_Event_Handler::windowFocusLost(const SDL_Event *event) {
 }
 
 void SDL_Event_Handler::windowResized(const SDL_Event *event) {
-  enigma_user::draw_clear(enigma_user::window_get_color());
+  graphics_remove_garbage(0,0,enigma_user::window_get_width(),enigma_user::window_get_height());
```

Comment:
You will need to stub this function somewhere for graphics systems direct x 9/11 and none